src/main/java/com/google/codeu/data/User.java

```diff
@@ -0,0 +1,20 @@
+package com.google.codeu.data;
+
+public class User {
+
+    private String email;
+    private String aboutMe;
+
+    public User(String email, String aboutMe) {
+        this.email = email;
+        this.aboutMe = aboutMe;
```

Comment:
Along with the code formatting, adding another nullness check here might make sense.